photon-client/src/main/scala/com/linkedin/photon/ml/cli/game/GameDriver.scala

```diff
@@ -186,13 +186,14 @@ trait GameDriver extends PhotonParams {
 
     val shardConfigs = getRequiredParam(featureShardConfigurations)
     val allFeatureSectionKeys = shardConfigs.values.map(_.featureBags).reduce(_ ++ _)
-    val nameAndTermFeatureSetContainer = NameAndTermFeatureSetContainer.readNameAndTermFeatureSetContainerFromTextFiles(
+    val nameAndTermFeatureMap = NameAndTermFeatureMapUtils.readNameAndTermFeatureMapFromTextFiles(
```

Comment:
The `RDD` objects referenced by `nameAndTermFeatureMap` should be persisted, as they may be used multiple times each on line 196.